components/TextEditor/TextEditorContainer.module.scss

```diff
@@ -2,9 +2,12 @@
 .editorBox {
   border-style: solid;
   border-width: 5px;
+  width: available;
+  max-width: fit-content;
```

Comment:
I _think_ if we set the width of the child editor to be 100%, you should just be able to set the width of this to be the width you actually want the box to be on the page, and everything should be fine

Reply:
Child editor won't take percentages as width or height. Doing so makes it disappear. Every other way I try to solve this results in black space on the right except "fit-content".

Reply:
Bea got `width: 100%` working on `admin` so it should work fine.
But you will need to give the parent a fixed width (otherwise 100% of nothing is still nothing)

Reply:
Of course! Done